shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue';
+
+declare global {
+  namespace JSX {
```

Comment:
wozu ist diese Datei

Reply:
TypeScript braucht Typdefinitionen für nicht-TypeScript-Code, in diesem Fall Vue templates und JSX… JSX/TSX (JavaScript mit Unterstützung für inline-HTML ohne `"`) nutzen wir im Moment nicht.